popde/adaptive_kde.py

```diff
@@ -82,6 +82,41 @@ def set_per_point_bandwidth(self, pilot_values):
         self.set_bandwidth(
             self.global_bandwidth / self._local_bandwidth_factor(pilot_values)
         )
+    
+    def set_alpha(self, new_alpha):
+        """
+        Set the adaptive parameter alpha to a new value.
+
+        Parameter:
+            new_alpha (float): The new value for the adaptive parameter alpha.
+        """
+        self.alpha = new_alpha
+        # compute pilot_values
+        self.pilot_values = self.evaluate(self.kde_data)
```

Comment:
do the pilot values actually need to be recalculated here?  I think they do not depend on alpha (only on global bw)

Reply:
They are needed here to update bandwidth

Reply:
yes, I know you need to _use_ pilot values, but I am saying that `self.pilot_values` already exists and you do not have to recalculate them